<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KInstitutionPage</class>
 <widget class="QWidget" name="KInstitutionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>322</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>-1</number>
   </property>
   <property name="margin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="qLabel1">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string>If this account is managed by an institution please select the institution from the list. If the institution does not exist yet, please choose the &lt;b&gt;New Institution&lt;/b&gt; button to create it. Otherwise, leave this field empty.
Enter the account number used by the institution to identify the account.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel4">
         <property name="text">
          <string>Institution</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KComboBox" name="m_institutionComboBox">
         <property name="toolTip">
          <string>Select the institution which manages this account or leave empty</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="m_newInstitutionButton">
         <property name="text">
          <string>New Institution</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="textLabel5">
         <property name="text">
          <string>Account number</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KLineEdit" name="m_accountNumber">
         <property name="whatsThis">
          <string>Accounts managed by an institution are usually referenced by a unique number the so called account number. Enter that number here.

KMyMoney currently uses this number only for some online banking functions.</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>IBAN</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="KLineEdit" name="m_iban">
         <property name="toolTip">
          <string>Enter the International Bank Account Number into this field</string>
         </property>
         <property name="whatsThis">
          <string>You can enter the IBAN (International Bank Account Number) into this field if you know it. Usually, people at your institution can tell you the number or it is printed on your statements.

See also http://en.wikipedia.org/wiki/International_Bank_Account_Number for more information.

KMyMoney keeps this field only for documentation purposes and does not use it otherwise.</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer16">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>132</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer17">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
